discover/analytics-cloud/articles/01-what-is-liferay-analytics-cloud.markdown

```diff
@@ -0,0 +1,124 @@
+# What is Liferay Analytics Cloud?
+
+Your company interacts with lots of people, including existing and prospective
+customers. On your websites they browse products and services, ask questions,
+leave feedback, etc. Presenting relevant, interesting website content motivates
+people to do business with you. Knowing your customers well and understanding
+how they interact with existing digital content can help you improve their
+experience. Liferay Analytics Cloud generates powerful, intuitive charts that
+deepen your knowledge of people and their online behavior so you can create
+terrific user experiences that drive business. 
+
+It all starts with understanding people.
+
```

Comment:
I like this! My only feedback is maybe we can say something more high level about Analytics, rather than going right to charts. Something like this: 

Liferay Analytics Cloud deepens your knowledge of how people use your site, so you can create terrific user experiences that drive business and make your visitors happier.